### PR TITLE
Point the closing comment with Pull request link

### PR DESCRIPTION
**Pull request by shreekarSS**
Signed-off-by: Shreekara <sshreeka@redhat.com>

## Related Issues and Dependencies
Fixes: https://github.com/thoth-station/kebechet/issues/1039
…

### CODE REVIEW

**Comment by KPostOffice** [comment]
What happens with overlays when there are multiple PRs associated with a single run?

File: kebechet/managers/update/update.py [hunk 1]
```diff
@@ -922,6 +915,28 @@ def run(self, labels: list = []) -> Optional[dict]:
...
+                        issue.comment(
+                            CLOSE_MANUAL_ISSUE_COMMENT_PR.format(
+                                pr=pr,
+                                time=datetime.now().strftime("%m/%d/%Y, %H:%M:%S"),
+                            ),
+                        )
+                    issue.close()
+
+            else:
+                close_manual_update_issue = partial(
```
Comment (KPostOffice) [inline kebechet/managers/update/update.py:932]: This doesn't need to be a partial as it is only called once. Creating partial functions is useful when you have a function getting called with many of the same parameters throughout a piece of code. It allows you to create an intermediate function that has certain parameters bound.

Comment (shreekarSS) [reply kebechet/managers/update/update.py:932]: ack

File: kebechet/managers/update/update.py [hunk 2]
```diff
@@ -922,6 +915,26 @@ def run(self, labels: list = []) -> Optional[dict]:
                             )
                         result = {}
                 results[self.runtime_environment] = result
-            close_manual_update_issue()
+
+            if overlays_dir:
+                issue = self.get_issue_by_title(_ISSUE_MANUAL_UPDATE)
+                if issue:
```
Comment (KPostOffice) [inline kebechet/managers/update/update.py:921]: These should be the outer conditional that way the `if issue` is checked for both conditional branches

**Review by KPostOffice — APPROVED** [review]

**Comment by goern** [comment]
is this released and deployed?

**Comment by shreekarSS** [comment]
> is this released and deployed?

I'll get update on release and revert

**Pull request merged**